src/app/clusters/scenes/SceneTableImpl.cpp

```diff
@@ -197,40 +254,78 @@ struct FabricSceneData : public PersistentData<kPersistentFabricBufferMax>
         return writer.EndContainer(fabricSceneContainer);
     }
 
-    CHIP_ERROR Deserialize(TLV::TLVReader & reader) override
+    /// @brief This Deserialize method is implemented only to allow compilation. It is not used throughout the code.
+    /// @param reader TLV reader
+    /// @return CHIP_NO_ERROR
+    CHIP_ERROR Deserialize(TLV::TLVReader & reader) override { return CHIP_NO_ERROR; }
```

Comment:
This should return an error, just in case someone calls it.  Maybe `CHIP_ERROR_INCORRECT_STATE`?